webpack/components/extensions/about/SystemStatuses.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { translate as __ } from 'foremanReact/common/I18n';
+import API from '../../../services/api';
+
+class SystemStatuses extends Component {
+  constructor(props) {
+    super(props);
+    this.pingServices = this.pingServices.bind(this);
+    this.state = { services: {} };
+  }
+  componentDidMount() {
+    this.pingServices();
+  }
+
+  pingServices() {
+    API.get('/ping').then(({ data: { services } }) => {
+      this.setState({ services });
```

Comment:
Why storing data in state rather than using store? I think we should also show spinner when loading.

Reply:
+1 to this.